site/content/installation/running-on-kind.md

```diff
@@ -40,13 +40,59 @@ To install NGINX Gateway Fabric, choose the appropriate installation guide that
 - [Installation with Helm]({{< relref "installation/installing-ngf/helm.md" >}})
 - [Installation with Kubernetes manifests]({{< relref "installation/installing-ngf/manifests.md" >}})
 
+## Set up a nodeport
+
+When using kind clusters, be aware that NodePort services require [additional setup](https://kind.sigs.k8s.io/docs/user/configuration/#nodeport-with-port-mappings).
+
+For example, the following will automatically set up port forwarding into a local cluster (mostly for development):
+
+```
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 31437
+    hostPort: 8080
+    protocol: TCP
+  - containerPort: 31438
+    hostPort: 4443
```

Comment:
Can we change this to `8443` to be consistent with the ports we use for https traffic?

Reply:
Resolved.